lib/code_corps/github/sync/sync.ex

```diff
@@ -38,10 +38,11 @@ defmodule CodeCorps.GitHub.Sync do
 
   [https://developer.github.com/v3/activity/events/types/#issuesevent](https://developer.github.com/v3/activity/events/types/#issuesevent)
   """
-  def issue_event(%{"issue" => issue} = payload) do
+  @spec issue_event(map) :: outcome
+  def issue_event(%{"issue" => issue_payload} = payload) do
     Multi.new
-    |> Multi.merge(__MODULE__, :find_repo, [payload])
```

Comment:
We had a `find_repo/2` which was made public explicitly so we could use `Multi.merge/4`. That indicates to me we should not use it and instead use `Multi.merge/2`. It reduces code fragmentation, and makes this function easier to understand.

Same for other similar cases below.

Reply:
Yes. Sorry.